publishable/config/voyager.php

```diff
@@ -1,6 +1,46 @@
 <?php
 
 return [
+
+    /*
+    |--------------------------------------------------------------------------
+    | Factory config
+    |--------------------------------------------------------------------------
+    |
+    | Here you can specify what classes should be returned when using e.g Voyager::model('Category')
+    |
+    */
+
+    'factory' => [
+        'models' => [
+            'Category'   => TCG\Voyager\Models\Category::class,
+            'DataRow'    => TCG\Voyager\Models\DataRow::class,
+            'DataType'   => TCG\Voyager\Models\DataType::class,
+            'Menu'       => TCG\Voyager\Models\Menu::class,
+            'MenuItem'   => TCG\Voyager\Models\MenuItem::class,
+            'Page'       => TCG\Voyager\Models\Page::class,
+            'Permission' => TCG\Voyager\Models\Permission::class,
+            'Post'       => TCG\Voyager\Models\Post::class,
+            'Role'       => TCG\Voyager\Models\Role::class,
+            'Setting'    => TCG\Voyager\Models\Setting::class,
+            'User'       => TCG\Voyager\Models\User::class,
+        ],
+    ],
+
+    /*
+    |--------------------------------------------------------------------------
+    | Policies config
+    |--------------------------------------------------------------------------
+    |
+    | Here you can specify policies for certain models
+    |
+    */
+
+    'policies' => [
```

Comment:
These two policies were defined in `VoyagerServiceProvider` because they don't have BREAD to set the policy on (see `VoyagerServiceProvider::registerGates()`).  I think moving this to a config, while normally s good idea, will lead to confusion with Voyager users.  Instead of making the change through the UI, they'll do it through config.  This may be a better practice for most apps, but is contrary to the purpose of Voyager.

Reply:
The reason behind this was as noted above for overridability, if i don't use the set class for these policies it would not work as expected. I could fix this setting by being more dynamic in the provider instead, which might be a better Idea actually. to avoid confusion but still keep it dynamic. I'll fix it ASAP!